code/drasil-example/Drasil/DblPendulum/Body.hs

```diff
@@ -31,116 +55,118 @@ mkSRS = [RefSec $      --This creates the Reference section of the SRS
-  -- --     --  , PhySysDesc dblpendulum physSystParts figLaunch []
-  -- --       , Goals goalsInputs]
-  --        SSDSolChSpec $ SCSProg
-  --        [ Assumptions
-  --          , TMs [] (Label : stdFields)
-  --       , GDs [] ([Label, Units] ++ stdFields) ShowDerivation
-  --       , DDs [] ([Label, Symbol, Units] ++ stdFields) ShowDerivation
-  --     --  , IMs [] ([Label, Input, Output, InConstraints, OutConstraints] ++ stdFields) ShowDerivation
-  --     --  , Constraints EmptyS inConstraints
-  --     --  , CorrSolnPpties outConstraints []
-  --      ]
-  --    ],
+  IntroSec $            -- This adds an introduction with an overview of the sub-sections
+    IntroProg justification (phrase pendulumTitle) -- This adds an introductory blob before the overview paragraph above.
+      [IScope scope],                            -- This section add a Scope section with the content of 'scope' constructor.
+  SSDSec $ 
+    SSDProg                               -- This adds a Specific system description section and an introductory blob.
+      [ SSDProblem $ PDProg prob []                --  This adds a is used to define the problem your system will solve
+        [ TermsAndDefs Nothing terms               -- This is used to define the terms to be defined in terminology sub section
+      , PhySysDesc pendulumTitle physSystParts figMotion [] -- This defines the Physicalsystem sub-section, define the parts
+                                                          -- of the system using physSysParts, figMotion is a function in figures for the image
+      , Goals goalsInputs] -- This adds a goals section and goals input is defined for the preample of the goal.
+      , SSDSolChSpec $ SCSProg --This creates the solution characteristics section with a preamble
+        [ Assumptions -- This adds the assumption section
+        , TMs [] (Label : stdFields)
+        , GDs [] ([Label, Units] ++ stdFields) ShowDerivation
+        , DDs [] ([Label, Symbol, Units] ++ stdFields) ShowDerivation
+        , IMs [] ([Label, Input, Output, InConstraints, OutConstraints] ++ stdFields) ShowDerivation
+ --       , Constraints EmptyS inConstraints
+ --       , CorrSolnPpties outConstraints []
+       ]
+     ],
   --ReqrmntSec $
   --  ReqsProg
   --     [ FReqsSub EmptyS []
   --     , NonFReqsSub
   --     ],
   -- TraceabilitySec $ TraceabilityProg $ traceMatStandard si,
-    AuxConstntSec $
-     AuxConsProg dblpendulum [],  --Adds Auxilliary constraint section
+  AuxConstntSec $
+     AuxConsProg pendulumTitle [],  --Adds Auxilliary constraint section
   Bibliography                    -- Adds reference section
   ]
 
--- justification, scope :: Sentence
--- justification = foldlSent [S "dblpendulum" +:+. S "dblpendulum",
---   phrase dblpendulum]
--- scope = foldlSent_ [S "the", (`sOf` S "a"), phrase dblpendulum]
+justification :: Sentence
+justification = foldlSent [S "pendulumTitle is the subject" +:+. S "pendulumTitle is the focus",
```

Comment:
`pendulumTitle` in the string? I think this should be fixed before merging.

Reply:
done